### ISSUE TITLE
[BUG] export data only export single table

### ISSUE THREAD

**Issue by u007** [cweijan/vscode-database-client#397]
hi,

ive right click on my connected db and click on export data,
but the result sql only show single table.

- OS: PopOS 20.04
- Database Type: MySQL

here is the full file:
```
-- MySQL dump 10.13  Distrib 8.0.27, for Linux (x86_64)
--
-- Host: 127.0.0.1    Database: ajinorik_db1
-- ------------------------------------------------------
-- Server version	5.7.36

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8mb4 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `deliveries`
--

DROP TABLE IF EXISTS `deliveries`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `deliveries` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `user_id` varchar(255) DEFAULT NULL,
  `description` varchar(255) DEFAULT NULL,
  `created_at` datetime DEFAULT NULL,
  `updated_at` datetime DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `deliveries`
--

/*!40000 ALTER TABLE `deliveries` DISABLE KEYS */;
INSERT INTO `deliveries` VALUES (1,NULL,'aaa','2014-05-24 07:17:53','2014-05-24 07:17:53'),(2,NULL,'','2014-06-03 13:37:02','2014-06-03 13:37:02'),(3,NULL,'','2014-06-03 13:37:10','2014-06-03 13:37:10');
/*!40000 ALTER TABLE `deliveries` ENABLE KEYS */;
```

**Comment by cweijan** [comment]
This file does not look complete, try backing up again.

**Comment by u007** [comment]
already tried many times, same result

**Comment by cweijan** [comment]
Does your backup file have this end?
![image](https://user-images.githubusercontent.com/27798227/145512068-4e334709-22b9-456e-adc8-2799270e1371.png)

**Comment by cweijan** [comment]
The documents you listed are incomplete, and this is unlikely to happen.

**Comment by u007** [comment]
ya dont have the completed
thats exactly the content of the file

**Comment by u007** [comment]
here is the recording...

https://user-images.githubusercontent.com/388813/145530277-c2471b4c-d082-4750-9a34-cf1dbee83b3d.mp4

**Comment by cweijan** [comment]
Thank you for the recording, I will try to find the cause of the problem。

**Comment by u007** [comment]
okay i think i got this error:
```
mysqldump: [Warning] Using a password on the command line interface can be insecure.
mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
mysqldump: Couldn't execute 'SELECT COLUMN_NAME,                       JSON_EXTRACT(HISTOGRAM, '$."number-of-buckets-specified"')                FROM information_schema.COLUMN_STATISTICS                WHERE SCHEMA_NAME = 'ajinorik_db1' AND TABLE_NAME = 'friendly_id_slugs';': Unknown table 'COLUMN_STATISTICS' in information_schema (1109)
```

**Comment by u007** [comment]
i did some search found its due to mysql8 client
https://serverfault.com/questions/912162/mysqldump-throws-unknown-table-column-statistics-in-information-schema-1109

**Comment by cweijan** [comment]
This means that you use a lower version of mysql_dump to back up a higher version of mysql, you need to remove the old version of mysql_dump from the environment variable and restart vscode.

**Comment by u007** [comment]
but eventually alot of distribution will move to mysql8 by default

**Comment by cweijan** [comment]
MySQL dump 10.13 comes with mysql5.7, it does not support mysql8.

**Comment by u007** [comment]
hmm weird, is there a way to configure a setting to add a flag --column-statistics=0 to the mysqldump?

**Comment by cweijan** [comment]
Thanks, adding this option does work, I have released the new version 4.3.5.

**Comment by u007** [comment]
tested working :100:
